app/src/utils/utils.ts

```diff
@@ -62,6 +62,12 @@ export function daysAgo(val = 0) {
   return formatter.format(Math.round(deltaDays), 'days');
 }
 
+// Convert unix ts to DD-MM-YYYY
+export function formatDate(dateStr: BigNumber): string {
+  const date = new Date(BigNumber.from(dateStr).toNumber() * 1000);
+  return `${date.getDate()}-${date.getMonth()}-${date.getFullYear()}`;
```

Comment:
@apbendi + @gdixon this is what i was talking about

Reply:
`getMonth()` is zero indexed and returns a value between 0 through 11

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth

Reply:
UGH FML

Reply:
```suggestion
  return `${date.getDate()}-${date.getMonth()+1}-${date.getFullYear()}`;
```

Reply:
😜😂

Reply:
Months range from 0-11 in javascript (a quirk that was copied from java)! So we just need to +1 on the month (with an exception for when adding 1 will incr the year too)!